iarc_main/scripts/follow.py

```diff
@@ -24,9 +25,20 @@ def run(self):
 
 	            # Plan the motion
 	            cmd = Twist()
-	            cmd.linear.x = position[0]
-	            cmd.linear.y = position[1]
-	            cmd.angular.z = 0
+
+	            # maxvelocity of our drone
+	            maxvelocity = 1
```

Comment:
consider `maxvelocity = 1.0` to make it clear that this can be a floating point value.

Even better, `maxvelocity = rospy.get_param('~max_velocity', 1.0)` using the [ROS parameter system](http://wiki.ros.org/rospy_tutorials/Tutorials/Parameters).